contractcourt/channel_arbitrator_test.go

```diff
@@ -148,8 +171,41 @@ func (*mockChainIO) GetBlock(blockHash *chainhash.Hash) (*wire.MsgBlock, error)
 	return nil, nil
 }
 
-func createTestChannelArbitrator(log ArbitratorLog) (*ChannelArbitrator,
-	chan struct{}, chan []ResolutionMsg, chan *chainntnfs.BlockEpoch, error) {
+type chanArbTestCtx struct {
+	chanArb *ChannelArbitrator
+
+	cleanUp func()
+
+	resolvedChan chan struct{}
+
+	blockEpochs chan *chainntnfs.BlockEpoch
+
+	incubationRequests chan struct{}
+
+	resolutions chan []ResolutionMsg
+}
+
+func (c *chanArbTestCtx) CleanUp() {
+	c.cleanUp()
+}
+
+func (c *chanArbTestCtx) ResolvedContracts() chan struct{} {
+	return c.resolvedChan
```

Comment:
Why the accessor functions and now using the chan directly?

Reply:
Started to go in a slightly more elaborate direction, but stopped short. Will modify to just use the channels directly.